packages/core/src/shared/extensionGlobals.ts

```diff
@@ -96,39 +98,61 @@ function resolveGlobalsObject(): ToolkitGlobals {
 function proxyGlobals(globals_: ToolkitGlobals): ToolkitGlobals {
     return new Proxy(globals_, {
         get: (target, prop) => {
+            // Test for initialize()
+            if (Object.keys(target).length === 0) {
+                throw new Error(`ToolkitGlobals accessed before initialize()`)
+            }
+
+            // Test that the property was set before access.
+            // Tradeoff: not being able to do something like `globals.myValue ??= ...` without a try/catch
```

Comment:
Good point, useful comment. But for globals, assigning with `??=` is probably a sign of buggy startup logic...